linera-service/src/cli_wrappers.rs

```diff
@@ -1077,3 +1078,65 @@ impl NodeService {
         serde_json::from_value(data["requestApplication"].clone()).unwrap()
     }
 }
+
+pub struct ApplicationWrapper<A> {
+    uri: String,
+    _phantom: PhantomData<A>,
+}
+
+impl<A> ApplicationWrapper<A> {
+    pub async fn raw_query(&self, query: impl AsRef<str>) -> Value {
```

Comment:
Since this is not necessary only used in test code (think future consumers of this code) it would be good to have proper error handling here and return a `Result` without the unwraps.

Reply:
I'd like to postpone that to a later PR:
There are already 27 `unwrap()` calls in `cli_wrappers` and most of the code in this module is actually only used in tests. We should probably make some of it test-only again.

Reply:
Unwrap gives more debuggable errors. We probably should use lots of .context(..) if we switch all to ?

Reply:
I kept the `unwraps`/`expect`s for now, but feature-gated the test-only code: https://github.com/linera-io/linera-protocol/pull/1124